lib/Element.js

```diff
@@ -233,6 +237,14 @@ Element.prototype.getElementById = function (id) {
   }
 }
 
+Element.prototype.getBoundingClientRect = function () {
+  if (!this.component) {
+    return undefined
+  }
+
+  return this.component.getBoundingClientRect()
+}
```

Comment:
Interesting, I'm not sure how this would work to be honest. The bounding rect is something computed after the browser has rendered the element and applied the CSS. Would you be able to test this? Thanks for the contribution.